snapcraft.yaml

```diff
@@ -942,6 +945,9 @@ parts:
       - --prefix=/usr
       - --with-builtin=pulse
     build-environment: *buildenv
+    override-build: |
```

Comment:
I did some digging, can we ensure config.guess is updated upstream?

Reply:
Lennart Poettering has not released a new version of [libcanberra](http://0pointer.de/lennart/projects/libcanberra/) since 2012. This project looks ophaned.

Reply:
This thread give some background: https://discourse.gnome.org/t/is-libcanberra-deprecated/10336